operatorconfig/driverconfig/powermax/v2.10.1/node.yaml

```diff
@@ -99,7 +99,7 @@ spec:
             - name: X_CSI_POWERMAX_ENDPOINT
               value: "<X_CSI_POWERMAX_ENDPOINT>"
             - name: X_CSI_K8S_CLUSTER_PREFIX
-              value: "<X_CSI_K8S_CLUSTER_PREFIX>"
+              value: "CSM"
```

Comment:
why are we defaulting to CSM in the older versions? It should be only done in the latest version i.e. `2.12.0`